File: fixtures/scale/swift/large/Families/Rot13/Rot13Runner.swift
// Correctness entry point for the rot13 family; wraps CorrectnessEngine
// with this family's cipher and known-answer vectors.
struct Rot13Runner {
    private let engine = CorrectnessEngine()

    func check() -> VectorOutcome {
        let cipher = Rot13Cipher()
        let vectors = Rot13Vectors.all()
        return engine.verify(cipher, vectors)
    }
}

File: fixtures/scale/php/large/src/Families/Rot13/Rot13Runner.php
<?php

namespace CipherLab\Families\Rot13;

use CipherLab\Abstractions\VectorOutcome;
use CipherLab\Engine\CorrectnessEngine;

// Correctness entry point for the rot13 family; wraps CorrectnessEngine
// with this family's cipher and known-answer vectors.
class Rot13Runner
{
    private CorrectnessEngine $engine;

    public function __construct()
    {
        $this->engine = new CorrectnessEngine();
    }

    public function check(): VectorOutcome
    {
        return $this->engine->verify(new Rot13Cipher(), Rot13Vectors::all());
    }
}
